### PR TITLE
Fix/#252

### PR DESCRIPTION
**Pull request by DoubleCouponDay**
Implements request #252.

### CODE REVIEW

File: src/pages/about.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ const AboutPage = () => (
 
           <h2>Past and Present Contributors</h2>
 
-          <div className="cards-list">
+          <div className="cards-list contributor-cards">
```
Comment (williamparry) [inline src/pages/about.js]: Can this be fixed with the classes we have?

Comment (DoubleCouponDay) [reply src/pages/about.js]: There needs to be some difference between the founder cards and the contributor cards and currently they share the same css classes. I don't think it's possible unless someone has a unique strategy.

Comment (DoubleCouponDay) [reply src/pages/about.js]: Resolved by using margin.

**Comment by DoubleCouponDay** [comment]
Closed due to not up to date. Will reopen a new branch.

**Pull request closed without merging**